packages/configraph/src/config-node.ts

```diff
@@ -297,12 +297,13 @@ export class ConfigraphNode<NodeMetadata = any> {
     // now deal with resolution
     // TODO: need to track dependencies used in coerce/validate/etc
 
-    // TODO: probably want to _skip_ resolution if we have overrides present?
-    const itemResolverCtx = new ResolverContext(this.valueResolver || this);
+    const itemResolverCtx = new ResolverContext((!this.overrides.length && this.valueResolver) || this);
     resolverCtxAls.enterWith(itemResolverCtx);
 
-
-    if (this.valueResolver) {
+    if (this.overrides.length) {
```

Comment:
previously we were still running the resolver, and would still surface resolution errors if there were any. This now skips the resolution instead - both removing the error, and potentially speeding things up.